sprayjson/src/main/scala/io/finch/sprayjson/package.scala

```diff
@@ -29,4 +29,9 @@ package object sprayjson {
     */
   implicit def encodeSpray[A](implicit format: JsonFormat[A]): Encode.Json[A] =
     Encode.json((a, cs) => BufText(a.toJson.prettyPrint, cs))
+
+  implicit val encodeExceptionSpray: Encode.Json[Exception] =
```

Comment:
And to say more, in the decodeSpray I think we can safely replace JsonFormat with JsonReader.